src/Symfony/Bundle/FrameworkBundle/Resources/views/Collector/http_client.html.twig

```diff
@@ -0,0 +1,80 @@
+{% extends '@WebProfiler/Profiler/layout.html.twig' %}
+
+{% block toolbar %}
+    {% if collector.traces|length %}
+        {% set icon %}
+            {{ include('@WebProfiler/Icon/memory.svg') }}
+            {% set status_color = '' %}
+            <span class="sf-toolbar-value">{{ collector.traces|length }}</span>
+        {% endset %}
+
+        {{ include('@WebProfiler/Profiler/toolbar_item.html.twig', { link: profiler_url, status: status_color }) }}
+    {% endif %}
+{% endblock %}
+
+{% block menu %}
+<span class="label {{ collector.traces is empty ? 'disabled' }}">
+    <span class="icon">{{ include('@WebProfiler/Icon/memory.svg') }}</span>
+    <strong>HTTP Client</strong>
+</span>
+{% endblock %}
+
+{% block panel %}
+    <h2>HTTP Requests</h2>
+    {% for trace in collector.traces %}
+        <table>
+            <thead>
+                <tr>
+                    <th colspan="2">{{ trace.request.method }} {{ trace.request.url }}</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <th>Method</th>
+                    <td>
+                        {{ trace.request.method }}
+                    </td>
+                </tr>
+                <tr>
+                    <th>URL</th>
+                    <td>
+                        {{ trace.request.url }}
+                    </td>
+                </tr>
+                <tr>
+                    <th>Options</th>
+                    <td>
+                        {{ trace.request.options is empty ? 'none' : profiler_dump(trace.request.options, maxDepth=1) }}
+                    </td>
+                </tr>
+            </tbody>
+            <tbody>
+                <thead>
+                    <tr>
+                        <th colspan="2">Response</th>
+                    </tr>
+                </thead>
+                <tbody>
+                <tr>
+                    <th>Status Code</th>
+                    <td>
+                        {% if trace.response.statusCode >= 500 %}
+                            {% set responseStatus = 'error' %}
+                        {% elseif trace.response.statusCode >= 400 %}
```

Comment:
Why is a `4xx` response only worth a warning? The request was unsuccessful, after all.

Reply:
Server error versus client error. I'm agree with @jeremyFreeAgent on that point

Reply:
That is used for debug purpose. I think as a developer you want to know if your request is bad. So `400` should be as a warning IMO.

Reply:
My application is the client. So if I receive a `400` error, my application did something wrong. This is an error and should be flagged as such.

I understand that you want to separate server from client errors somehow and I agree that we should do that. But the concept of warning/error does not fit here, imho.

Reply:
Perhaps `error` for `4XX` and `warning` for `5XX`?

Reply:
Let's go with the current colors and iterate later.